spec.html

```diff
@@ -329,19 +329,32 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-default-method-descriptor" aoid="DefaultMethodDescriptor">
-    <h1>DefaultMethodDescriptor ( key, closure, enumerable, placement )</h1>
+  <emu-clause id="sec-default-method-property-descriptor" aoid="DefaultMethodPropertyDescriptor">
+    <h1>DefaultMethodPropertyDescriptor ( key, value, enumerable )</h1>
     <emu-alg>
-      1. Perform SetFunctionName(_closure_, _key_).
+      1. Assert: _key_ is a Private Name, or ! IsPropertyKey(_key_) is *true*.
+      1. Assert: Type(_enumerable_) is Boolean.
       1. If _key_ is a Private Name,
         1. Set _enumerable_ to *false*.
         1. Let _configurable_ be *false*.
         1. Let _writable_ be *false*.
-        1. If _placement_ is `"prototype"`, set _placement_ to `"own"`.
       1. Else,
         1. Let _configurable_ be *true*.
         1. Let _writable_ be *true*.
-      1. Let _desc_ be the PropertyDescriptor{[[Value]]: _closure_, [[Writable]]: _writable_, [[Enumerable]]: _enumerable_, [[Configurable]]: _configurable_}.
+      1. Return the PropertyDescriptor{[[Value]]: _value_, [[Writable]]: _writable_, [[Enumerable]]: _enumerable_, [[Configurable]]: _configurable_}.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-default-method-descriptor" aoid="DefaultMethodDescriptor">
+    <h1>DefaultMethodDescriptor ( key, closure, enumerable, placement )</h1>
+    <emu-alg>
+      1. Assert: ! IsCallable(_closure_) is *true*.
+      1. Assert: _key_ is a Private Name, or ! IsPropertyKey(_key_) is *true*.
+      1. Perform ! SetFunctionName(_closure_, _key_).
+      1. If _key_ is a Private Name,
+        1. Let _writable_ be *false*.
+        1. Assert: _placement_ is `"own"`.
```

Comment:
I don't understand this. Where does "prototype" change into "own" for private methods? Also, what about static private?

Reply:
Ah, maybe I need clarification here.

When would the placement be "prototype" in the first place for a private method?

Reply:
Well, whenever you use instance method syntax. That's why the previous spec text was changing prototype into own.

Reply:
See https://tc39.github.io/proposal-decorators/#sec-method-definitions-runtime-semantics-propertydefinitionevaluation

Reply:
aha, thanks. in that case i'll update this to be correct :-)